src/components/Dashboard/TodoList.vue

```diff
@@ -1379,6 +1415,12 @@ export default class TodoList extends Vue {
         this.$router.push({ name: Routes.STANDALONE_ADDRESSES, params: { filingId: item.filingId.toString() } })
         break
 
+      case FilingTypes.CONSENT_CONTINUATION_OUT:
+        // resume this Consent to Continuation Out locally
+        this.setCurrentFilingStatus(FilingStatus.DRAFT)
+        this.$router.push({ name: Routes.CONSENT_CONTINUATION_OUT, params: { filingId: item.filingId.toString() } })
+        break
+
```

Comment:
Similar comment as above.  We're extending an existing pattern and this is not the appropriate time to refactor, but I'd suggest the TodoList component is strong candidate for refactoring.  Perhaps it's just me, but multiple `if-then-else` or `case` statements based on the same criteria (in this case FilingType) is a hint that there may be a cleaner way to structure this code.

Reply:
Agreed -- the Todo List is a strong candidate for refactoring. I thought a bit about it a while back and wanted to do "something" to make it easier to have unit tests (esp. to prevent regressions). I started some unit test changes along those lines but didn't have time to finish it.